docs/configuration-reference/components/rook-ceph.md

```diff
@@ -71,6 +73,7 @@ Example:
 |-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|:-------:|:--------:|
 | `namespace`             | Namespace to deploy the Ceph cluster into. Must be the same as the rook operator.                                                                  | rook    | false    |
 | `monitor_count`         | Number of Ceph monitors to deploy. An odd number like 3 or 5 is recommended which should also be sufficient for most cases.                        | 1       | false    |
+| `enable_toolbox`        | Install toolbox pod to debug and manage the Ceph Cluster.                                                                                          | false   | false    |
```

Comment:
Could you please fix the missing article and the capitalization here? Also, I'd use "deploy" instead of "install" since this is the term we use in the other knobs.